app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -406,7 +406,8 @@ def process_member_changes
 
         def add_members_to_collection(collection = nil)
           collection ||= @collection
-          collection.add_member_objects batch
+          error_messages = collection.add_member_objects batch
+          flash[:notice] = error_messages.join(" ") if error_messages.any?
```

Comment:
Rather than manually populating the flash here, I would try the following instead:

`collection.errors.add(:collections, error_messages) if error_messages.any?`

And rely on code higher up to detect errors and render the right flash message.

Reply:
also, why `notice`? Typically errors should be in the `:error` flavor of  flash message.